force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +239,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List <Integer> result = new List<Integer> ();
 
         // Add the integers 1, 2, and 3 to the list
         // add 1
         // add 2
         // add 3
-
+        result.add(1);
+        result.add(2);
+        result.add(3);
```

Comment:
Try adding the numbers in the constructor instead of after the list is created
https://salesforce.stackexchange.com/questions/203694/how-to-initialize-list-set-values-by-construction-in-apex